examples/hello/group_vars/secret/patient0/development/edxapp/credentials.vault.yml

```diff
@@ -0,0 +1,42 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
An exemple credentials need to be vaulted ?

Reply:
You're right, there's no secrets files required for hello. Removed them all.